ang/civiawards/award-creation/directives/review-panels/review-panel-popup.html

```diff
@@ -120,7 +120,33 @@
         <div class="form-group">
           <label class="col-sm-6 control-label">Anonymise Contacts for this Panel?</label>
           <div class="col-sm-3">
-            <input type="checkbox" ng-model="model.currentReviewPanel.visibilitySettings.anonymizeApplication">
+            <input type="checkbox"
+              ng-model="model.currentReviewPanel.visibilitySettings.anonymizeApplication">
```

Comment:
Just a question: Shouldn't we use the civicase checkbox throughout civicase/awards/etc ?

Reply:
`civicase checkbox` is only used in the Manage Case/Dashboard screens, outside we are not using it. Because for other pages, we are still using the CiviCRM look and feel, and not the CiviCase.

Reply:
thanks for the clarification